J/JuliaInterpreter/Compat.toml

```
@@ -1,20 +1,39 @@
-["0-0.7"]
-julia = "1"
+["0 - 0.7"]
+julia = "1.0"
+# assuming exact match was wanted
+# otherwise, use ~ or ^ specifier
 
-["0.2-0.7.13"]
-CodeTracking = "0.3.2-0.5"
+["0.2 - 0.7.13"]
+CodeTracking = "0.3.2 - 0.5"
+# note this allows
+# CodeTracking [0.3.2 - 0.6)
+# otherwise use
+# CodeTracking = "0.3.2 - 0.5.8"
 
-["0.7.14-0.7.20"]
-CodeTracking = "0.5.9-0.5"
+["0.7.14 - 0.7.20"]
+CodeTracking = "0.5.9 - 0.5"
 
-["0.7.21-0"]
-CodeTracking = ["0.5.9-0.5", "1"]
+["0.7.21 - 0"]
+CodeTracking = "0.5.9 - 0.5-1"
+# unclear what is meant here
+# or if this is a mistake
+# CodeTracking 0.5.9 to 1 or to 0.5.10?
+# CodeTracking latest version is now at 1.0.8
 
-["0.8-0.8.4"]
-julia = ["1.0", "1.5.0-1"]
+["0.8 - 0.8.4"]
+julia = "1.0 - 1.5.0"
```

This is _**not**_ the same as before.  Plus, we want to prevent it from being installable on v1.8
```suggestion
 julia = ["1.0", "1.5.0-1.7"]
```